<launch>

	<!-- Joystick node -->
	<node name="joy_node" pkg="joy" type="joy_node" output="screen" respawn="false">
		<param name="dev" type="str" value="/dev/input/js0"/>
	</node>

    <!-- IARRC joystick node -->
	<node name="iarrc_joystick" pkg="iarrc" type="iarrc_joystick" output="screen" respawn="false">
		<param name="joystick_topic" type="str" value="/joy"/>
		<param name="speed_topic" type="str" value="/speed"/>
		<param name="steering_topic" type="str" value="/steering"/>
		<param name="angle_max" type="int" value="20"/>
		<param name="speed_max" type="int" value="20"/>
	</node>

</launch>
